docs/h5py_dataset.rst

```diff
@@ -55,11 +55,17 @@ things if your data happens to meet these assumptions:
 * All data is stored into a single HDF5 file.
 * Data sources reside in the root group, and their names define the source
   names.
-* Data sources are not explicitly split. Instead, splits are defined as
-  attributes of the root group. They're expected to be numpy arrays of
-  shape ``(2,)``, with the first element being the starting point
-  (inclusive) of the split and the last element being the stopping
-  point (exclusive) of the split.
+* Data sources are not explicitly split. Instead, splits are defined in the
```

Comment:
I would say, "not explicitly split into separate HDF5 datasets or separate HDF5 files."